examples/example-react-task/craco.config.ts

```diff
@@ -21,17 +21,26 @@ import { CracoConfig } from "@craco/types";
 const config: CracoConfig = {
   webpack: {
     configure(config, context) {
-      config.experiments = {
-        topLevelAwait: true,
+      const experiments = {
         ...config.experiments,
-      };
+        topLevelAwait: true,
+      }
 
-      return config;
+      const resolve = {
+        ...config.resolve,
+        fallback: {
+          crypto: require.resolve('crypto-browserify'),
+          stream: require.resolve('stream-browserify'),
```

Comment:
Interesting, what errors did you run in to when these were not included? And did you run the app directly in the JS repo or did you grab just the example app directory?

Reply:
Thanks for the repro steps Nick. Still can't reproduce these errors unfortunately but will keep trying tmrw.

Reply:
Update: As per our Slack discussion, we'll remove `pnpm` instructions from the Docs and encourage only `npm` as of now.